CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 * [#162](https://github.com/python-qt-tools/PyQt5-stubs/pull/162) fixes all method not declared as signals
 * [#184](https://github.com/python-qt-tools/PyQt5-stubs/pull/184) Fix missing module variable
    detected by latest mypy 0.930
-
+* [#183](https://github.com/python-qt-tools/PyQt5-stubs/pull/183) Add missing operations on QSize
+   
```

Comment:
```suggestion

```

Reply:
Oups, probably a side effect of web merging.